monome/teletype
# Fixes #118 by eliminating TEMP from flash storage

#### What does this PR do?

Fixes nvram_data_t to remove TEMP script from storage.

#### How should this be manually tested?

Save some flash scenes and load them again.  Ensure that scripts have been copied properly.

#### Any background context you want to provide?

2.1 introduced this bug when TEMP_SCRIPT was added.

#### I have,
* [na] updated the documentation
